src/test/java/fi/aalto/cs/apluscourses/utils/DirAwareZipFileTest.java

```diff
@@ -1,79 +0,0 @@
-package fi.aalto.cs.apluscourses.utils;
-
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.doCallRealMethod;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-
-import java.io.FileNotFoundException;
-import java.util.ArrayList;
-import java.util.List;
-import net.lingala.zip4j.exception.ZipException;
-import net.lingala.zip4j.model.FileHeader;
-import org.junit.Test;
-
-public class DirAwareZipFileTest {
```

Comment:
Info for other reviewers: I asked Nikolas to delete this test (instead of trying to "fix" it) written by me because it was bad, and shouldn't have been there in the first place.